### ISSUE TITLE
Firefox Exceptions

### ISSUE THREAD

**Issue by VackarAfzal** [handsontable/ngHandsontable#87]
Hi,

I'm currently using an older version if this library, but would really like to use the new validation feature you added.  However running the latest build seems to be throwing exceptions with Firefox.
```
TypeError: event.path is undefined
```

Issue seems to be in handontable.full.js:4593
```
len = event.path.length;
```
Any ideas on what's causing it? If you could fix this it woud be amazing, I could lend a had too if needed.

Many Thanks,
Vackar

**Comment by budnix** [comment]
Hi, this should be fixed in this release 0.6.0.

**Comment by VackarAfzal** [comment]
Thanks, I'll check this out now!
